.gitlab-ci.yml

```diff
@@ -79,12 +79,21 @@ test:unit:
       - unit-coverage.tar
 
 publish:tests:
-  image: alpine
+  image: golang:1.14
   stage: publish
-  before_script:
-    - apk add --no-cache bash curl findutils git
+  # See https://docs.coveralls.io/parallel-build-webhook
+  variables:
+    COVERALLS_WEBHOOK_URL: "https://coveralls.io/webhook"
   dependencies:
```

Comment:
Also:

* This is unused (it would only be used for the parallel-build hook which this pipeline is not using).

* A `before_script` is missing with `CI_BRANCH` and `CI_PR_NUMBER` env variables. These are important for coveralls to link correctly back to GitHub. See mender and mender-artifact for reference.